packages/xsnap/src/replay.js

```diff
@@ -1,6 +1,6 @@
 /**
  * Replay usage:
- *   node -r esm replay.js <folder>...
```

Comment:
I think that breaks usage of this module as a CLI script, powered by `if (require.main === module) { ... }` at the bottom.

NESM doesn't have an "if I have been invoked as a CLI script ..." idiom, does it?

Reply:
Evidently `new URL(import.meta.url).pathname === process.argv[1]` is the new idiom, or near to it as I could find.

Reply:
Yeah I almost went down that rabbit hole, and apparently there is a package that wrap that logic (with some extra extension handling): https://www.npmjs.com/package/es-main